apps/admin/backend/src/cast_vote_records.ts

```diff
@@ -285,6 +286,10 @@ export async function importCastVoteRecords(
       const votes = convertCastVoteRecordVotesToTabulationVotes(
         castVoteRecordCurrentSnapshot
       );
+      const adjudicationFlags = getCastVoteRecordAdjudicationFlags(
+        votes,
+        electionDefinition
+      );
```

Comment:
What happens if the votes are later changed by write-in adjudication?

Reply:
@mattroe and I discussed, and decided that we are not handling that for now. If a user wanted to see a report on blank ballots, they _probably_ are more interested in the number of blank ballots before adjudication rather than after. The latter case we would probably meet with another set of flags, when we come to that.

Reply:
Ok, do we want to make that clearer in the UI and/or in the code?

Reply:
I added a comment, but I don't think this is important enough to make explicit in the UI. It's still a bit of a corner case since we only have write-in adjudication and not general purpose adjudication.

- Overvote status cannot change, because we cannot yet adjudicate overvotes
- Blank ballot and write-in status could _arguably_ change if we validate a UWI, but not worth worrying about this NH edge case
- The undervote status can change significantly with write-in adjudication, so that could be confusing. But I'd wager that is the least sought after report anyway.

There's a case for having the post-adjudication status eventually and, at that point, we'll have to find a way to be clear about it. But for now, not an issue IMO.

Reply:
related discussion: https://votingworks.slack.com/archives/CEL6D3GAD/p1699381326254249